CODEOWNERS

```diff
@@ -89,6 +89,7 @@ Kconfig*                                  @tejlmand
 /samples/profiler/                        @pdunaj @pizi-nordic
 /samples/peripheral/radio_test/           @kapi-no
 /samples/peripheral/lpuart/               @nordic-krch
+/samples/tfm_hello_world/                 @oyvindronningstad
```

Comment:
Should this be in `samples/subsys/tfm/tfm_hello_world` ?

Reply:
Will move to samples/tfm/tfm_hello_world